homeassistant/components/zwave_js/number.py

```diff
@@ -0,0 +1,82 @@
+"""Support for Z-Wave controls using the number platform."""
+from typing import Callable, List, Optional
+
+from zwave_js_server.client import Client as ZwaveClient
+
+from homeassistant.components.number import DOMAIN as NUMBER_DOMAIN, NumberEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import DATA_CLIENT, DATA_UNSUBSCRIBE, DOMAIN
+from .discovery import ZwaveDiscoveryInfo
+from .entity import ZWaveBaseEntity
+
+
+async def async_setup_entry(
```

Comment:
Please include a test for this platform.

Reply:
I do not have mock data for this. I needed to add it in because of the light platform discovery change but I don't have a device with valve control, which is currently the only discovered entity in the scheme. I could create a fake device...

Reply:
There is a dump here: https://github.com/home-assistant/core/issues/45703#issuecomment-771721491

node 4

Reply:
Thanks. I've added a test based on that dump.

Reply:
test added